eZ/Publish/SPI/Repository/Values/MultiLanguageDescriptionValueAware.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Repository\Values;
+
+/**
+ * This is the interface for all ValueObjects implementing translated description logic.
+ *
+ * Provides a uniform way for API consuming logic to generate translated description labels
+ * for API objects.
+ */
+interface MultiLanguageDescriptionValueAware
```

Comment:
naming.. `Aware` has specific meaning in symfony, so lets perhaps avoid that. Also as it's a interface and not a base value class like Value.php, we can skip imo that to.
So mayme rather just`MultiLanguageDescription`, wdyt @Nattfarinn ?